tank/main.py

```diff
@@ -11,15 +11,19 @@
 from tank.core.exc import TankError
 from tank.controllers.base import Base
 from tank.controllers.cluster import NestedCluster, EmbeddedCluster
+from tank.logging_conf import build_logging_conf
+
+
+logger = logging.getLogger(__name__)
 
 
 def _default_config() -> Dict:
     config = init_defaults('tank',
                            'log.logging')
 
     config['tank'] = {
-        'terraform_run_command': '/usr/local/bin/terraform',
-        'terraform_inventory_run_command': '/usr/local/bin/terraform-inventory',
+        'terraform_run_command': 'terraform',
```

Comment:
Will it work? Are they on $PATH?
I suggest installing into subdir `bin` of `app.user_dir` and adding this subdir to the `PATH` at `app_env`

Reply:
@Eenae I unpack archive to `app.user_dir` then I add this directory to `PATH` variable.

Reply:
Should I unpack it to `bin` or remain it as it is?

Reply:
lets use `bin`

Reply:
@Eenae done